docs/guides/administrator/troubleshooting.md

```diff
@@ -59,3 +59,56 @@ When the Opstrace controller (a deployment running in the Opstrace cluster) is s
 kubectl logs deployment/opstrace-controller \
   --all-containers=true --namespace=kube-system > controller.log
 ```
+
+## Known Issues
+
+It's possible your installation can fail waiting for the certificates to be ready. When this happens, you'll see messages such as the following:
+
+```text
+info: waiting for 0 DaemonSets
+info: waiting for 0 StatefulSets
+info: waiting for 1 Certificates
+debug:   Waiting for Certificate ingress/https-cert to be ready
+```
+
+And when the installation fails:
+
+```text
+info: waiting for 0 DaemonSets
+info: waiting for 0 StatefulSets
+info: waiting for 1 Certificates
+warning: cluster creation attempt timed out after 2400 seconds
+error: 3 attempt(s) failed. Stop retrying. Exit.
+```
+
+This is a known problem that we are tracking in these issues:
+
+* [ci: waiting "for Certificate ingress/https-cert to be ready" didn't resolve](https://github.com/opstrace/opstrace/issues/151)
+* [Certificate sometimes fails to issue properly](https://github.com/jetstack/cert-manager/issues/3594)
+
+If this happens, the recommended workaround is to restart the certificate request process.
+
+First, if the installation process is stuck, you can exit by pressing Ctrl-C. Then proceed to [connect kubectl to your Opstrace cluster](#kubernetes-based-debugging).
+
+Afterward, find the certificate request created by cert-manager:
+
+```bash
+$ kubectl -n ingress get certificaterequest
+NAME                         READY   AGE
+https-cert-XXXXX             False   30m
+kubed-apiserver-cert-XXXXX   True    30m
+```
+
+The certificate request starts with `https-cert-` followed by five random characters.
+
+Delete the failed certificate request:
+
+```bash
+kubectl -n ingress delete certificaterequest https-cert-XXXXX
+```
+
+And delete the certificate to have the controller recreate it and restart the request process:
+
+```bash
+kubectl -n ingress delete certificate http-cert
+```
```

Comment:
How does one know that it was now successful?  Re-launch the CLI?

Reply:
Added instructions to check it was successful.